src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindPaymentAccountsByEmailTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.professionalapi.domain.ContactInformation;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.domain.PaymentAccount;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+
+import uk.gov.hmcts.reform.professionalapi.persistence.*;
+
+import uk.gov.hmcts.reform.professionalapi.util.ProfessionalReferenceDataClient;
+import uk.gov.hmcts.reform.professionalapi.util.Service2ServiceEnabledIntegrationTest;
+
+public class FindPaymentAccountsByEmailTest extends Service2ServiceEnabledIntegrationTest {
+
+    @Autowired
+    private OrganisationRepository organisationRepository;
+    @Autowired
+    private ProfessionalUserRepository professionalUserRepository;
+    @Autowired
+    private ContactInformationRepository contactInformationRepository;
+    @Autowired
+    private DxAddressRepository dxAddressRepository;
+    @Autowired
+    PaymentAccountRepository paymentAccountRepository;
+
+    private ProfessionalReferenceDataClient professionalReferenceDataClient;
+
+    @Before
+    public void setUp() {
+        professionalReferenceDataClient = new ProfessionalReferenceDataClient(port);
+        dxAddressRepository.deleteAll();
+        contactInformationRepository.deleteAll();
+        professionalUserRepository.deleteAll();
+        paymentAccountRepository.deleteAll();
+        organisationRepository.deleteAll();
+    }
+
+    @Test
+    public void get_request_returns_correct_payment_accounts() {
+        Organisation newOrganisation = new Organisation("some-org-", "pending", "sra-id", "comp-num", false, "company-url");
+        Organisation theOrganisation = organisationRepository.save(newOrganisation);
+
+        PaymentAccount thePaymentAcc = new PaymentAccount("pbaNum");

```

Comment:
1. Can you please reuse builder methods available for organisation creation rather than duplicating the code.
2 . Remove system.out.println method from test class and make sure before committing remove such statements.

Reply:
I have included these changes in my latest commit. Please review. Thanks!